static/bib/yoshitom.bib

```diff
@@ -1,3 +1,30 @@
+@misc{yoshitom-levorato-2020-icpr,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Neural Compression and Filtering for Edge-assisted Real-time Object Detection in Challenged Networks},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.15818},
+  primaryclass = {cs.CV},
+}
+
+@misc{yoshitom-levorato-2020-emdl,
+  author = {Yoshitomo Matsubara and Marco Levorato},
+  title = {Split Computing for Complex Object Detectors: Challenges and Preliminary Results},
+  year = {2020},
+  archiveprefix = {arXiv},
+  code = {https://github.com/yoshitomo-matsubara/hnd-ghnd-object-detectors},
+  eprint = {2007.13312},
+}
```

Comment:
Perhaps it is a good idea to make it clear that it is an arXiv preprint
(which, I assume, we will replace with a proper journal publication
once it will be released):

```suggestion
  note = {arXiv preprint},
}
```